src/platform/nxp/k32w/k32w0/GroupKeyStoreImpl.h

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2020 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Provides an implementation of the Chip Group Key Store interface
+ *          for platforms based on the NXP SDK.
+ */
+
+#include <core/CHIPKeyIds.h>
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+#include <platform/nxp/k32w/k32w0/K32W0Config.h>
+
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+
+class ChipGroupKey
+{
+public:
+    enum
+    {
+        MaxKeySize = 36
+    };
+    uint32_t KeyId;          /**< The key ID. */
+    uint8_t KeyLen;          /**< The key length. */
+    uint8_t Key[MaxKeySize]; /**< The secret key material. */
+    union
+    {
+        uint32_t StartTime; /**< The epoch key start time. */
+        uint32_t GlobalId;  /**< The application group key global ID. */
+    };
+};
+
+/**
+ * An implementation of the Chip GroupKeyStoreBase API for platforms based
+ * on the NXP SDK.
+ */
+class GroupKeyStoreImpl final : public ::chip::Profiles::Security::AppKeys::GroupKeyStoreBase, private K32WConfig
```

Comment:
is this  supposed to be in this PR? Seems independent of a layout change

Reply:
Hi @andy31415 , thanks for the comment.
No, GroupKeyStoreImpl.h should not be in this PR, I see the file has been deleted in master by the time I pushed the commit.
Removed it from the project.